tests/arrows/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -245,88 +419,609 @@ fooooooooooooooooooooooooooooooooooooooooooooooooooo(action => next =>
 
 `;
 
-exports[`comment.js 1`] = `
-/**
- * Curried function that ends with a BEM CSS Selector
- *
- * @param {String} block - the BEM Block you'd like to select.
- * @returns {Function}
- */
-export const bem = block =>
-  /**
-   * @param {String} [element] - the BEM Element within that block; if undefined, selects the block itself.
-   * @returns {Function}
-   */
-  element =>
-    /**
-     * @param {?String} [modifier] - the BEM Modifier for the Block or Element; if undefined, selects the Block or Element unmodified.
-     * @returns {String}
-     */
-    modifier =>
-      [
-        ".",
-        css(block),
-        element ? \`__\${css(element)}\` : "",
-        modifier ? \`--\${css(modifier)}\` : ""
-      ].join("");
-
-<FlatList
-  renderItem={(
-    info, // $FlowExpectedError - bad widgetCount type 6, should be Object
-  ) => <span>{info.item.widget.missingProp}</span>}
-  data={data}
-/>
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-/**
- * Curried function that ends with a BEM CSS Selector
- *
- * @param {String} block - the BEM Block you'd like to select.
- * @returns {Function}
- */
-export const bem = block =>
-  /**
-   * @param {String} [element] - the BEM Element within that block; if undefined, selects the block itself.
-   * @returns {Function}
-   */
-  element =>
-    /**
-     * @param {?String} [modifier] - the BEM Modifier for the Block or Element; if undefined, selects the Block or Element unmodified.
-     * @returns {String}
-     */
-    modifier =>
-      [
-        ".",
-        css(block),
-        element ? \`__\${css(element)}\` : "",
-        modifier ? \`--\${css(modifier)}\` : ""
-      ].join("");
+exports[`call.js 2`] = `
+Seq(typeDef.interface.groups).forEach(group =>
+  Seq(group.members).forEach((member, memberName) =>
+    markdownDoc(
+      member.doc,
+      { typePath: typePath.concat(memberName.slice(1)),
+       signatures: member.signatures }
+    )
+  )
+)
 
-<FlatList
-  renderItem={(
-    info // $FlowExpectedError - bad widgetCount type 6, should be Object
-  ) => <span>{info.item.widget.missingProp}</span>}
-  data={data}
-/>;
+const promiseFromCallback = fn =>
+  new Promise((resolve, reject) =>
+    fn((err, result) => {
+      if (err) return reject(err);
+      return resolve(result);
+    })
+  );
 
-`;
+runtimeAgent.getProperties(
+  objectId,
+  false, // ownProperties
+  false, // accessorPropertiesOnly
+  false, // generatePreview
+  (error, properties, internalProperties) => {
+    return 1
+  },
+);
 
-exports[`currying.js 1`] = `
-const fn = b => c => d => {
-  return 3;
-};
+function render() {
+  return (
+    <View>
+      <Image
+        onProgress={(e) => this.setState({progress: Math.round(100 * e.nativeEvent.loaded / e.nativeEvent.total)})}
```

Comment:
Should we treat JSX expression containers the same way we treat template literals and function calls, and omit the parens when possible (when using the default option)?